src/management/apps.ts

```diff
@@ -330,3 +342,114 @@ export async function getAppConfig(appId: string, platform: AppPlatform): Promis
   }
   return response.body;
 }
+
+/**
+ * Lists all Firebase android app SHA certificates identified by the specified app ID.
+ * @param projectId the project to list SHA certificates for.
+ * @param appId the ID of the app.
+ * @return list of all Firebase android app SHA certificates.
+ */
+export async function listAppAndroidSha(
+  projectId: string,
+  appId: string
+): Promise<AppAndroidShaData[]> {
+  const shaCertificates: AppAndroidShaData[] = [];
+  try {
+    const response = await api.request(
+      "GET",
+      `/v1beta1/projects/${!appId ? projectId : "-"}/androidApps/${!appId ? "-" : appId}/sha`,
```

Comment:
I think we should always pass projectId here, is there a case where `-` works?

Reply:
Also as discussed earlier `appId` is also required (sorry for the misinformation before)

Reply:
in a quantum parallel universe i believe that you and me have already accepted it as a future feature to support listing only by projectId

Reply:
done

Reply:
😆